ritm_annotation/core/annotation/__init__.py

```
@@ -0,0 +1,18 @@
+
+from .session import AnnotationSession
+from .events import AnnotationEvent, EventType
+from .state import AnnotationState, ObjectState
+
+__all__ = [
+    'AnnotationSession',
+    'AnnotationEvent',
+    'EventType',
+    'AnnotationState',
+    'ObjectState',
+]
```

Ruff (0.14.3)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)